test/miner_state_channel_SUITE.erl

```diff
@@ -0,0 +1,546 @@
+-module(miner_state_channel_SUITE).
+
+-include_lib("common_test/include/ct.hrl").
+-include_lib("eunit/include/eunit.hrl").
+-include_lib("kernel/include/inet.hrl").
+-include_lib("blockchain/include/blockchain_vars.hrl").
+-include("miner_ct_macros.hrl").
+
+-export([
+         init_per_suite/1,
+         end_per_suite/1,
+         init_per_testcase/2,
+         end_per_testcase/2,
+         all/0
+        ]).
+
+-export([
+         no_packets_expiry_test/1,
+         packets_expiry_test/1,
+         multi_clients_packets_expiry_test/1,
+         replay_test/1
+        ]).
+
+%% common test callbacks
+
+all() -> [
+          no_packets_expiry_test,
+          packets_expiry_test,
+          multi_clients_packets_expiry_test,
+          replay_test
+         ].
+
+init_per_suite(Config) ->
+    Config.
+
+end_per_suite(Config) ->
+    Config.
+
+init_per_testcase(_TestCase, Config0) ->
+    Config = miner_ct_utils:init_per_testcase(?MODULE, _TestCase, Config0),
+    Miners = ?config(miners, Config),
+    Addresses = ?config(addresses, Config),
+    Balance = 5000,
+    InitialPaymentTransactions = [ blockchain_txn_coinbase_v1:new(Addr, Balance) || Addr <- Addresses],
+    InitialDCTxns = [blockchain_txn_dc_coinbase_v1:new(Addr, Balance) || Addr <- Addresses],
+    AddGwTxns = [blockchain_txn_gen_gateway_v1:new(Addr, Addr, h3:from_geo({37.780586, -122.469470}, 13), 0)
+                 || Addr <- Addresses],
+
+    NumConsensusMembers = ?config(num_consensus_members, Config),
+    BlockTime = ?config(block_time, Config),
+    BatchSize = ?config(batch_size, Config),
+    Curve = ?config(dkg_curve, Config),
+
+    SCVars = #{max_open_sc => 2,
```

Comment:
Comments on these vars so we know what they mean

Reply:
Added